gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/TTimestampWrapper.java

```diff
@@ -132,4 +156,57 @@ public int getMibiecIntValue() {
     public int getAntcfinalIntValue() {
         return timestamp.getANTCFinal().getV().intValue();
     }
+
+    //
+
+    Map<String, Boolean> getExportCornerActiveForCountryMap() {
```

Comment:
All the "export corner" namings for variable that aims at knowing if a country is exporting or not towards Italy feels amibiguous to me. Why not rename something like exportToItaly ? To make things clearer. I think we should keep the "export corner" name for things that concern the specific process branch saying "some countries are importing from Italy" in a generic way.